docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +49,169 @@ described below, and relative to the joint TMREL.
+
+TMREL in GBD 2017
+-----------------
+For each of the country-derived relative risk surfaces, the 500 g and 2-week gestational age joint bin
+with the lowest risk was identified. This bin differed within each country dataset. To identify the
+universal 500 g and 2-week gestational age category that would serve as the universal TMREL, all bins that were identified as the TMREL was chosen. This is cat55 (40-42ga, 3500-400g) and cat56 (40-42ga, 4000-4500g)
+
+.. note::
+   the TMREL categories listed in GBD 2017 risk appendix is wrong.  
+
+Causes that are affected by LBWSG
+---------------------------------
+
++----------+---------------------------------------------------------+
+| Cause id | Cause                                                   | 
++==========+=========================================================+
+|  302     | diarrheal diseases                                      |
++----------+---------------------------------------------------------+
+|  322     | lower respiratory tract infections                      |
++----------+---------------------------------------------------------+
+|  328     | upper respiratory tract infections                      | 
++----------+---------------------------------------------------------+
+|  329     | otitis media                                            |
++----------+---------------------------------------------------------+
+|  333     | pneumococcal meningitis                                 |
++----------+---------------------------------------------------------+
+|  334     | H influenzae type B meningitis                          |
++----------+---------------------------------------------------------+
+|  335     | meningococcal meningitis                                |
++----------+---------------------------------------------------------+
+|  336     | other meningitis                                        |
++----------+---------------------------------------------------------+
+|  337     | encephalitis                                            |
++----------+---------------------------------------------------------+
+|  381     | neonatal preterm birth complications                    |
++----------+---------------------------------------------------------+
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | Neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
```

Comment:
It might be worthwhile to note here that there are several causes with CSMRs that are associated with birth weight, although they were excluded from GBD's risk-outcome pairs because of concerns with reverse causality (HIV, TB as examples I think).

Reply:
ok!